client/src/utils/utils.js

```diff
@@ -0,0 +1,38 @@
+export const getCartFlightsTotal = (cart) => {
+  let total = 0;
+  total += cart.flights.reduce(
+    (total, item) => item.departure.price + item.departure.taxes + total,
+    0,
+  );
+  total += cart?.flights.reduce(
+    (total, item) => item.arrival.price + item.arrival.taxes + total,
+    0,
+  );
+  return total;
+};
+
+export const getCartHotelTotal = (cart) => {
+  let total = 0;
+  total += cart.hotels.reduce(
+    (total, item) => item.price + item.taxes + total,
+    0,
+  );
+  return total;
+};
+
+export const getCartCarTotal = (cart) => {
+  let total = 0;
+  total += cart.rentalCar.reduce(
+    (total, item) => item.price + item.taxes + total,
+    0,
+  );
+  return total;
+};
```

Comment:
Same as above. Check out the docs on [`reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce). You don't need to do

```javascript
let total = 0;
total += ...
return total;
```